install.js

```diff
@@ -537,29 +532,27 @@ function saveConfig(configItems, callback) {
       }
     }
   }
-
+  // add some default values as these aren't set
   var config = {
     outputPlugin: 'adapt',
     dbType: 'mongoose',
     auth: 'local',
     root: process.cwd()
   };
   // copy over the input values
-  _.each(configItems, function(value, key) {
-    config[key] = value;
-  });
-  
+  Object.keys(configItems).forEach(key => config[key] = configItems[key]);
```

Comment:
My only comment would be that you said it might read better with an `Object.assign()`.